bugbug/commit_features.py

```diff
@@ -178,8 +180,21 @@ def __call__(self, commit, **kwargs):
 
 
 class files(object):
+    def fit(self, commits, min_freq=0.00003):
+        self.min_freq = min_freq
```

Comment:
Since this is an option, you can set it in __init__ (so each model can decide what it wants).